src/app/Library/Auth/AuthenticatesUsers.php

```diff
@@ -47,6 +48,10 @@ public function login(Request $request)
         }
 
         if ($this->attemptLogin($request)) {
+            if (config('backpack.base.setup_email_verification_routes', false)) {
+                $this->verifyUserBeforeLogin($request);
+            }
```

Comment:
We can move the conditional inside the function too, right? Also... the function name... since it does something important (logout) I would put that in the name. It doesn't just "verify"... it logs out and redirects.

```suggestion
            $this->logoutIfEmailNotVerified($request);
```

Reply:
Ehrm I don't like this. I mean, I like that you mention logout there too, but I don't like switching the conditional inside the function. 
If I would be reading this code I would assume that `logoutIfEmailNotVerified()` is something that's GOING to happen.

In reality it would only be applied if the verification routes are set. 

I think we should keep the conditional here, since the function name implies that something is going to happen. But then it doesnt.

Let me know.

Reply:
Hmmm... yeah you might be right. Having the conditional inside it may be deceiving. Ok, we keep the conditional out.